src/Main.hs

```diff
@@ -85,6 +88,8 @@ data Opts = Opts
   { optsToken :: AuthToken
   , optsOffline :: Bool
   , optsIgnoreCase :: Bool
+  , optsSearchAll :: Bool
+  , optsShowDate :: Bool
   , optsBy :: Maybe String
   , optsOrganisation :: ParamName Organisation
   , optsFlow :: ParamName Flow
```

Comment:
Yes, currently you need to specify a dummy flow, which is prepended as part of the needle.

Can you have optional positional parameters before mandatory ones? The current way is the best I could figure out but I agree it's not ideal.